macos/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+if (UNIX AND APPLE)
+	# enable bundle
+	set_target_properties(blobby PROPERTIES MACOSX_BUNDLE TRUE)
```

Comment:
should this option be set even if BUILD_MACOS_BUNDLE is OFF?

Reply:
You are right. Moreover the logic was twisted. Thanks!